docs/sections/pipeline_samples/tutorials/GenerateSentencePair.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Maybe we can start adding the versions for third-party libraries for easier maintainability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>